packages/monaco/src/browser/style/index.css

```diff
@@ -1,9 +1,9 @@
 /* FIXME when https://github.com/Microsoft/monaco-editor/issues/113#issuecomment-240406949 is fixed */
-.monaco-editor .inputarea {
+/* .monaco-editor .inputarea {
```

Comment:
Can you please split up your PR into to separate ones. Or just revert the `CODE_EDITOR_PRIORITY` related change. That's not the fix for #5230 I believe.

Reply:
it's my mistake ,I push that commit in the same branch